.github/workflows/build-test.yml

```diff
@@ -42,5 +42,6 @@ jobs:
       if: ${{ github.event_name == 'push' }}
       with:
         commit_message: Push new compiler binaries
+        push_options: '--force'
         file_pattern: packages/google-closure-compiler-*/compiler*
-        github_token: ${{ secrets.GITHUB_TOKEN }}
+        skip_dirty_check: true
```

Comment:
Should we move these independent platforms to different commits based on success of the previous? It would be a bit slower to run in succession, but the Github UI would show the correct status for each of the actions.

Reply:
Gotcha. I'll have to figure out how to do the testing in parallel, and after that, the pushing in series. Will merge this and try that in a separate PR once the repo is actually deployed.

Reply:
Done in #8.